application/database/db.py

```diff
@@ -289,7 +330,7 @@ def gap_analysis(self, name_1, name_2):
             return None, None
         base_standard, _, _ = self.driver.execute_query(
             """
-            MATCH (BaseStandard:Node {name: $name1})
+            MATCH (BaseStandard:Standard|Tool {name: $name1})
```

Comment:
what is a base standard?

Reply:
Copying this in terms of naming one is a base to compare against (if we have more compares)
![Screenshot 2023-09-15 at 15 39 54](https://github.com/OWASP/OpenCRE/assets/10814889/a2ec2380-25d4-4ea9-9368-998a68807edb)